docs/operators/call.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "@call"
+---
+
+The **@call** operator is used to reference an `@http` operator. It is useful when you have multiple fields that resolves from the same HTTP endpoint.
```

Comment:
This is actually not accurate. `call` is used to refer to the generated resolver on a field that's defined at the root query and mutation.

Reply:
We should describe why we need this. `call` operator reduces duplication of code. For example, let's say we wish to set the baseURL for a few `http` directives but not all. We should come up with more such examples here.